stable/firehose/values.yaml

```diff
@@ -52,6 +61,10 @@ telegraf:
           - "http://localhost:8086"
         database: "test-db"
         retention_policy: "autogen"
+      prometheus_remote_write:
+        enabled: false
+        url: "http://localhost:8088"
+        version: "0.1.0"
```

Comment:
```suggestion
        version: "0.1.0"
        # authorization: Bearer <token>
```

Reply:
do we use different types of authz? if not, we could put the `Bearer` prefix also into the template and accept only the token from values.

Reply:
This can vary depending on setup, for example cortex doesn't provide any auth but suggests to put a simple proxy infront and just send it the `x-scope-orgid` header. One can implement a basic http auth or a bearer token auth for the same. Hence wanted to keep this open.

Reply:
ah okay. if that API itself doesn't have fixed standard, then we should keep this flexible.